packages/primitives/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types"
+	},
+	"include": [ "src/**/*" ],
+	"references": [ { "path": "../element" } ]
```

Comment:
That's the only project reference we need 👍 

https://github.com/WordPress/gutenberg/blob/753978099e30cb141216f5146f0791f460fbb049/packages/primitives/package.json#L27-L29